core/server/master/src/main/java/alluxio/master/file/meta/AsyncUfsAbsentPathCache.java

```diff
@@ -58,8 +60,10 @@
   private final MountTable mMountTable;
   /** Paths currently being processed. This is used to prevent duplicate processing. */
   private final ConcurrentHashMap<String, PathLock> mCurrentPaths;
-  /** Cache of paths which are absent in the ufs. */
-  private final Cache<String, Long> mCache;
+  /** Cache of paths which are absent in the ufs, maps an alluxio path to a Pair
+   *  which is the sync time and the mount id.
+   */
+  private final Cache<String, Pair<Long, Long>> mCache;
```

Comment:
Does it make sense to add some metrics for the absent path cache?

Reply:
yes, but maybe in a separate PR?